wiki/Difficulties/osu!/Easy/en.md

```diff
@@ -9,34 +12,34 @@ And try to avoid these:
 
 -   stacked notes
 -   overlapping notes
--   [Kick slider](Kick_slider)
+-   Kick slider
```

Comment:
idk if you want to (or if it'll be _set-in-stone_) but `Short Repeating Sliders` are also known as the `Kick Slider`... the soon-to-be link will be `[Short Repeating Sliders](/wiki/Mapping_Techniques/Slider/Short_Repeating_Sliders/)`